andrew/euler/euler_5.py

```diff
@@ -0,0 +1,24 @@
+'''
+What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?
+
+'''
+
+den = int(input("What is the smallest number that is evenly divisible \
```

Comment:
What is den? It should be obvious the first time I see it, not when I see the other parts of the code.